### PR TITLE
Dev v1.14.0

### PR DESCRIPTION
**Pull request by gschmidt-usgs**
Peer review for v1.14.0 for reading GeoTiff files

### CODE REVIEW

File: raw_binary/tools/test_read_gtif.c [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+RETURN VALUE:
+Type = None
+
+NOTES:
+******************************************************************************/
+void usage ()
+{
+    printf ("test_read_ard parses the XML, reads the Tiff files, and writes "
+            "back out the GeoTiff test files to duplicate each band.\n\n");
+    printf ("usage: test_read_ard --xml=xml_filename\n");
```
Comment (raydittmeier) [inline raw_binary/tools/test_read_gtif.c]: I think the filename should be test_read_gtif.  "test_read_ard" appears 3 times in the usage.

File: raw_binary/tools/test_read_gtif.c [hunk 2]
```diff
@@ -0,0 +1,306 @@
+/*****************************************************************************
+FILE: test_read_ard.c
```
Comment (raydittmeier) [inline raw_binary/tools/test_read_gtif.c]: The filename is test_read_gtif.c

File: raw_binary/tools/test_read_gtif.c [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+Value           Description
+-----           -----------
+ERROR           Error processing the ARD product
+SUCCESS         No errors encountered
+
+NOTES:
+******************************************************************************/
+int main (int argc, char** argv)
+{
+    char FUNC_NAME[] = "test_read_ard"; /* function name */
```
Comment (raydittmeier) [inline raw_binary/tools/test_read_gtif.c]: The function name probably should be test_read_gtif

Comment (gschmidt-usgs) [reply raw_binary/tools/test_read_gtif.c]: Copy/paste error.  Fixed.

File: raw_binary/tools/test_read_gtif.c (hunk 1 above)
Comment (gschmidt-usgs) [reply raw_binary/tools/test_read_gtif.c]: Copy/paste error.  Fixed.

File: raw_binary/tools/test_read_gtif.c (hunk 2 above)
Comment (gschmidt-usgs) [reply raw_binary/tools/test_read_gtif.c]: Copy/paste error.  Fixed.

**Pull request merged**